exercises/word-count/bower.json

```diff
@@ -7,7 +7,7 @@
     "output"
   ],
   "dependencies": {
-    "purescript-prelude": "^2.5.0",
+    "purescript-prelude": "^3.0.0",
     "purescript-maps": "^2.1.2"
```

Comment:
You will need to update all of these to the latest versions, including `devDependencies`.